djangocph/views.py

```diff
@@ -10,8 +12,36 @@ class IndexView(FormView):
     form_class = forms.ContactForm
     success_url = '/'
 
+    def is_spam(self, form):
+        """
+        Is Akismet still a thing?
```

Comment:
Yes, it is AFAIK :)

Otherwise you can try my project stopforumspam - it somehow still works... https://github.com/benjaoming/django-stopforumspam